flytekit/core/launch_plan.py

```diff
@@ -305,7 +309,7 @@ def __call__(self, *args, **kwargs):
         if ctx.compilation_state is not None:
             inputs = self.saved_inputs
             inputs.update(kwargs)
-            return create_and_link_node(ctx, entity=self, interface=self.workflow.python_interface, **inputs)
+            return create_and_link_node(ctx, entity=self, **inputs)
```

Comment:
is it safe to remove the interface arg here? if a launch plan has fixed inputs does it always have the same interface as the reference workflow?

Reply:
Good question @katrogan! `create_and_link_node` currently infers the interface from `entity.python_interface` And the output of `LaunchPlan.python_interface` is just `self.workflow.python_interface`, so at least the logic should be exactly the same. If we foresee a difference between an entity's `python_interface` attribute, and the interface we need to use to create the `Node`, then we can just define a different attribute in the protocol. Afaik, that's not necessary yet, but I'll defer to y'all!

Reply:
yeah it's fine.  i think it'll stay like this